setup.py

```diff
@@ -106,7 +107,7 @@ def is_win():
 
 setup(
     name="metadrive-simulator",
-    python_requires='>=3.7, <=3.9',
+    python_requires='>=3.7, <3.10',
```

Comment:
Can it work on 3.11 too?

Reply:
Yes, it can work. Just tested it and updated the ```setup.py```